reactDashboard/src/components/ToolTips/ToolTip.css

```diff
@@ -11,11 +11,13 @@
 }
 
 #tooltip > .tooltip-inner {
-  background-color: #000000;
+  background-color: #000000d9;
```

Comment:
As far as I can tell this d9 is to make it slightly translucent, yes?

Reply:
Minor point and not suggesting a change (as IE is not a target for us) - IE and older versions of browsers don't support this. https://caniuse.com/?search=color%20rgba

Reply:
xxxxxxd9 is the same as opactity: 85% so I have just changed to that ✅

Reply:
Change to decimal :)